sky/data/storage.py

```diff
@@ -45,9 +45,12 @@
 _MAX_CONCURRENT_UPLOADS = 32
 
 _BUCKET_FAIL_TO_CONNECT_MESSAGE = (
-    'Failed to connect to an existing bucket {name!r}.\n'
-    'Please check if:\n  1. the bucket name is taken and/or '
-    '\n  2. the bucket permissions are not setup correctly.')
+    'Failed to connect to an existing bucket {name!r}. '
```

Comment:
```suggestion
    'Failed to access an existing bucket {name!r}. '
```
Trying to fix nonstandard word. E.g., Google "connect to an existing s3 bucket".